openshift/template.yaml

```diff
@@ -31,6 +31,22 @@ objects:
                   value: "${SYNC_MODE}"
                 - name: DRY_RUN
                   value: "${DRY_RUN}"
+                - name: SNYK_TOKEN
+                  valueFrom:
+                    secretKeyRef:
+                      name: snyk-secrets
+                      key: token
+                - name: SNYK_URL
+                  value: "${SNYK_URL}"
+                - name: SNYK_INGESTION_FORCE
+                  value: "${SNYK_INGESTION_FORCE}"
+                - name: SNYK_FEED_FETCH_DAYS
+                  value: "${SNYK_FEED_FETCH_DAYS}"
```

Comment:
A description against each environment variable will be helpful for Future Reference.

Reply:
Description is mentioned below, where the macros are defined. Please have a look and let me know if it needs more description or if its satisfactory